### PR TITLE
Draft of RC1 announcement

### PR DESCRIPTION
**Pull request by chancancode**
cc @dhh @rafaelfranca

### CODE REVIEW

**Comment by rafaelfranca** [comment]
:shipit:

File: _posts/2014-11-28-Rails-4-2-0-rc1-has-been-released.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+author: chancancode
+published: true
+date: 2014-11-28 10:00:00 -08:00
+---
+
+Happy Black Friday everyone!
+
+The Rails team is very excited to announce that [the first Release Candidate of Rails 4.2.0][gem]
+has been released today. For an overview of all the major changes, please refer
+to the [release notes](release-notes).
```
Comment (robin850) [inline _posts/2014-11-28-Rails-4-2-0-rc1-has-been-released.markdown]: You certainly meant to use brackets here for the `release-notes` reference, no ?

File: _posts/2014-11-28-Rails-4-2-0-rc1-has-been-released.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+We would like to thank everyone who have assisted our team in testing the beta
+and smoothing out all the rough edges. During the beta period, we have logged
+[over 1500 commits by over 60 contributors][commits] and [over 100 tickets and pull requests][tickets]
+on Github.
```
Comment (robin850) [inline _posts/2014-11-28-Rails-4-2-0-rc1-has-been-released.markdown]: Really nit-picky thing but s/Github/GitHub; Otherwise, great work! :-)

**Comment by chancancode** [comment]
Thanks @robin850

**Pull request merged**